src/globals/scss/_helper-mixins.scss

```diff
@@ -54,6 +54,11 @@
     box-shadow: 0 0 0 3px $color__blue-20;
     outline: 1px solid transparent;
   }
+
+  @if ($type == 'experimental') {
```

Comment:
This is naming the type of focus outline so instead of `@include focus-outline('border');` or  `@include focus-outline('blurred');`its `    @include focus-outline('experimental');`  I don't think its a 1-1 mapping for all components that's why I didn't use the feature flag. 🤔

Reply:
Thanks @alisonjoseph for the background! One option I can think of is adding an arg to `focus-outline`, which is an optional color. Another option I can think of is a different name - The new one seems to be a flavor of `blurred` one, though?

Reply:
@asudoh Since it was just the color that was changing I added a new token to both themes and got rid of the experimental focus outline. I also renamed the $text-disabled to just $disabled because we are using for things that aren't just text. Let me know your thoughts!

Reply:
:100: thank you @alisonjoseph for the update!